src/index.ts

```diff
@@ -1,12 +1,16 @@
 import { RadarNativeInterface } from "./@types/RadarNativeInterface";
+import {Platform} from "react-native";
 
 let module: RadarNativeInterface;
 
 module = require("./index.native").default;
 
 export default module;
 
-export { default as RadarRNWeb } from "./index.web";
+let RadarRNWeb =  Platform.OS === 'web'?  require("./index.web").default: {};
```

Comment:
Hmmm, I'd feel better about this solve if we had a RN Web project to try it on. Basically for all the projects we're aware of we're just removing the import.

Reply:
there's also an extra space before 'Platform'